OpenRA.Game/Settings.cs

```diff
@@ -20,23 +20,61 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchSettings
```

Comment:
IMO it doesn't.  You could define these somewhere else in the Game namespace, but it shouldn't be here.